pages.es/linux/waypipe.md

```diff
@@ -0,0 +1,12 @@
+# waypipe
+
+> Ejecuta remotamente aplicaciones gráficas bajo un compositor de Wayland.
```

Comment:
```suggestion
> Ejecuta remotamente aplicaciones gráficas bajo un compositor Wayland.
```
IMO it would be a bit better to put it in the previous way, because the "compositor Wayland" indicates that it is a compositor "of the Wayland type" (that follows the Wayland protocol).0

Reply:
`para` maybe?